ReactNativeVersion/App.js

```diff
@@ -30,6 +31,24 @@ export default class App extends React.Component {
     this._unsubscribe('isPredicting');
   }
 
+  componentDidMount() {
+    this.gyro$ = Rx.Observable.create(observer => {
+      this.gyroscopeSubscription = Gyroscope.addListener(data =>
+        observer.next(data)
+      );
+    });
+    this.accel$ = Rx.Observable.create(observer => {
+      this.accelerometerSubscription = Accelerometer.addListener(data =>
+        observer.next(data)
+      );
+    });
+
+    this.accel$.subscribe(data =>
+      this.setState({ accel: data, count: this.state.count + 1 })
+    );
+    this.accel$.subscribe(data => this.setState({ gyro: data }));
```

Comment:
should this be this.gyro$ ?

Reply:
btw do you think there is battery trade off by subscribing here ? we could also put the setState in the toggleSubscription like before, if the battery implication isn't much then shouldnt matter

Reply:
Good catch. Yeah we can move the `setState` over to the toggle if there's a significant battery impact. Having the subscription here means we'll be able to view accelerometer and gyro data even if we're not recording.